SwiftCompilerSources/Sources/SIL/FunctionConvention.swift

```diff
@@ -77,13 +77,18 @@ public struct FunctionConvention : CustomStringConvertible {
   /// Collection of LifetimeDependenceConvention indexed on the
   /// function parameter.
   public var resultDependencies: ResultDependencies? {
-    let deps = bridgedFunctionType.SILFunctionType_getLifetimeDependenceInfo()
-    if deps.empty() {
-      return nil
+    let bridgedDependencies = bridgedFunctionType.SILFunctionType_getLifetimeDependencies()
+    let dependencies = LifetimeDependencies(bridged: bridgedDependencies)
+    let targetIndex = parameters.count
+
+    for (_, dependence) in dependencies.enumerated() {
```

Comment:
You enumerate but don't use the index... looks like something's wrong here

Reply:
Fixed